setup.cfg

```
@@ -36,6 +36,8 @@ setup_requires =
 install_requires =
   pyvista>=0.30.1
   sunpy>=3.0.0
+  matplotlib>=3.1.0
+  scipy>=1.3.0
```

```suggestion
  sunpy[map]>=3.0.0
```
Hopefully should fix it.